README.md

```diff
@@ -40,7 +41,14 @@ opam update
 opam switch create 5.0.0
 ```
 
-Using `opam` you can now `pin` and install them as follows:
+Using `opam` you can install the two testing libraries by running one
```

Comment:
I’m just wondering: could this formulation suggest you should install only of them?
I’d suggest the following phrasing: “The two testing libraries are available as OPAM packages `qcheck-lin` and `qcheck-stm` in the standard repository, so they can be installed in the usual way:”

Reply:
Thanks - rephrased in e6d18a6